cocoa/src/toga_cocoa/widgets/multilinetextinput.py

```diff
@@ -14,9 +17,8 @@
 
 class TogaTextView(NSTextView):
     @objc_method
-    def touchBar(self):
-        # Disable the touchbar.
-        return None
```

Comment:
This was needed in the past to avoid a crash if the app ran on a MacBook with a Touch Bar; however, touchbars are no longer present on modern Macs, and in my testing on an older macBook with a Touch Bar, the bug no longer manifests.